classes/services/queryBuilders/IssueListQueryBuilder.inc.php

```diff
@@ -152,10 +153,11 @@ public function countOnly($enable = true) {
 	public function get() {
 		$this->columns[] = 'i.*';
 		$q = Capsule::table('issues as i')
-					->where('i.journal_id','=', $this->contextId)
-					->leftJoin('issue_settings as is', 'i.journal_id', '=', 'is.issue_id')
-					->orderBy($this->orderColumn, $this->orderDirection)
-					->groupBy('i.issue_id');
+					->where('i.journal_id','=', $this->contextId);
+
+		if (empty($this->countOnly)) {
```

Comment:
@nibou230, can you walk me through this change? I'm not sure why e.g. we'd be able to get away with removing that `issue_settings` join without losing functionality here, or what SQL syntax we're trying to avoid..

Reply:
I honnestly doesn't even know why the settings are included here in the query as it's not even used later on. Maybe in some hooks? I think it would be better to keep as simple as possible and let the hooks manage the query the way they need. We can also let the left join in the query by adding a the distinct keyword.

Reply:
@NateWr could probably answer this better than me!

Reply:
I think this is the same as was addressed in this comment: https://github.com/pkp/ojs/pull/1919#discussion_r196026095

(In short, it can probably be removed for now, though we'll likely be adding it back in as the API is fleshed out.)